feat(efs): support for new `elastic` throughputmode (#23200)
aws/aws-cdk: packages/@aws-cdk/aws-efs/README.md

});
```

> ⚠️ An Amazon EFS file system's performance mode can't be MaxIo while its throughputMode is ELASTIC.

Reviewer: ```suggestion
⚠️ An Amazon EFS file system's performance mode can't be MAX_IO when its throughputMode is ELASTIC.
```
